### PR TITLE
WD : Feat todo

### PR DESCRIPTION
**Pull request by devananth**
### Completed Todo Feature

**Files to be checked:**
    
**Components:**
    - Todo.jsx
 
**Contexts:**
    - DataContext.js
    
**Reducers:**
    - dataReducer.js

**Demo:**

todoFeature

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,13 +13,10 @@ function App() {
...
   useEffect(() => {
-    fetch(
-      `https://api.unsplash.com/photos/random/?client_id=XqByhaz8qx069KZ_7Kk528zhO_02RDftaH3ioB8y1No&&orientation=landscape&&query=mountains%20dark`
-    )
-      .then((res) => res.json())
-      .then((x) => {
-        console.log(x);
-        setImageUrl(x.urls.regular);
+    fetch(UNSPLASH_API_URL)
+      .then((response) => response.json())
```
Comment (KoushikKintali) [inline src/App.js:17]: whenever you write .then make sure u write .catch too, what if ur UNSPLASH_API_URL server is down?

**Review by KoushikKintali — CHANGES_REQUESTED** [review]
Please handle catch 
or u can start using es6 syntaxes

**Review by prathmeshjagtap — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (devananth) [reply src/App.js:17]: Thanks for pointing it out.

**Comment by devananth** [comment]
> Please handle catch or u can start using es6 syntaxes

Can you pls check now, I have added **catch** block

**Review by spraveenofficial — APPROVED** [review]

**Pull request merged**